muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/announcer/body/application/new-edit-announcement.tsx

```diff
@@ -260,37 +247,57 @@ class NewEditAnnouncement extends SessionStateComponent<NewEditAnnouncementProps
-        placeholder={this.props.i18n.text.get('plugin.announcer.editannouncement.title.label')}
-        value={this.state.subject} onChange={this.onSubjectChange} autoFocus={!!this.props.announcement}/>),
-      (<CKEditor key="4" width="100%" height="grow" configuration={ckEditorConfig} extraPlugins={extraPlugins}
-       onChange={this.onCKEditorChange}>{this.state.text}</CKEditor>)
-    ]
-      
+      (
+      <div className="container container--new-announcement-title">    
+       <div className="environment-dialog__form-element--wrapper">  
+         <div className="environment-dialog__form-element-label">{this.props.i18n.text.get('plugin.announcer.createannouncement.title.label')}</div>          
+         <input key="3" type="text" className="environment-dialog__input--title"          
+          value={this.state.subject} onChange={this.onSubjectChange} autoFocus={!!this.props.announcement}/>
+       </div>   
+      </div>
+      ),
+      (
+      <div className="container container--announcer-content">    
+        <div className="environment-dialog__form-element--wrapper">  
+          <div className="environment-dialog__form-element-label">{this.props.i18n.text.get('plugin.announcer.createannouncement.content.label')}</div>          
+          <CKEditor key="4" width="100%" height="grow" configuration={ckEditorConfig} extraPlugins={extraPlugins}
+         onChange={this.onCKEditorChange}>{this.state.text}</CKEditor>
+        </div>       
+      </div>
+      )
+    ]      
     let footer = (closeDialog: ()=>any)=>{
       return (          
-         <div className="jumbo-dialog__button-container">
-          {this.recovered ? <Button buttonModifiers="danger" onClick={this.clearUp} disabled={this.state.locked}>
-            {this.props.i18n.text.get('clear draft')}
-          </Button> : null}
-          <Button buttonModifiers={["warn", "standard-cancel"]} onClick={closeDialog} disabled={this.state.locked}>
-            {this.props.i18n.text.get('plugin.announcer.createannouncement.button.cancel')}
-          </Button>
-          <Button buttonModifiers="standard-ok" onClick={this.createOrModifyAnnouncement.bind(this, closeDialog)}>
+         <div className="environment-dialog__button-container">
+          <Button className="button button-dialog--execute" onClick={this.createOrModifyAnnouncement.bind(this, closeDialog)}>
             {this.props.i18n.text.get('plugin.announcer.createannouncement.button.send')}
           </Button> 
+          <Button className="button button-dialog--cancel" onClick={closeDialog} disabled={this.state.locked}>
+            {this.props.i18n.text.get('plugin.announcer.createannouncement.button.cancel')}
+          </Button>            
+          {this.recovered ? <Button className="button button-dialog--clear" onClick={this.clearUp} disabled={this.state.locked}>
```

Comment:
This seems to be wrong regarding BEM. Button and button-dialog are both BEM blocks and it seems they are used as mixed blocks but button-dialog block has only modifier classname which is wrong.

Either use button or button-dialog only and then add modifier with base block name ie. "button-dialog button-dialog--execute".